rhel/SPECS/dbaas-controller.spec

```diff
@@ -0,0 +1,67 @@
+%undefine _missing_build_ids_terminate_build
+%global _dwz_low_mem_die_limit 0
+
+# do not strip debug symbols
+%global debug_package     %{nil}
+
+%global provider          github
+%global provider_tld      com
+%global project           percona-platform
+%global repo              dbaas-controller
+%global provider_prefix   %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path       %{provider_prefix}
```

Comment:
this is total mess. why do we need such amount of variables?

Reply:
No

Reply:
No, _we don't need them_ :)

Reply:
Removed these variables